### PR TITLE
HW20-22 ansible

### PR DESCRIPTION
**Pull request by dmitry231187**

### CODE REVIEW

File: ansible/roles/install_poetry/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  become: true
+  stat:
+    path: /home/{{ user_name }}/git
+  register: repo_result
+- name: Clone git repo
+  become: true
+  git:
+    repo: "{{ git_clone_url }}"
+    dest: /home/authz/git
+    version: master
```
Comment (ggramal) [inline ansible/roles/install_poetry/tasks/main.yml]: Lets introduce a variable here with which we can set code for what branch we will deploy

Comment (dmitry231187) [reply ansible/roles/install_poetry/tasks/main.yml]: introduced

File: ansible/roles/install_poetry/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    path: /home/{{ user_name }}/git
+  register: repo_result
+- name: Clone git repo
+  become: true
+  git:
+    repo: "{{ git_clone_url }}"
+    dest: /home/{{ user_name }}/git
+    version: "{{ git_branch }}"
+    force: yes
+- name: Poetry project install
```
Comment (ggramal) [inline ansible/roles/install_poetry/tasks/main.yml:13]: Have an error here in systemd logs
```
Jun 29 10:39:33 tms bash[25768]: Using python3.11 (3.11.4)
Jun 29 10:39:33 tms bash[25768]: Command not found: flask
```
service fails to start

Comment (ggramal) [reply ansible/roles/install_poetry/tasks/main.yml:13]: Seems like poetry install does not run

Comment (dmitry231187) [reply ansible/roles/install_poetry/tasks/main.yml:13]: maybe it was missed for some reason, let's try now

**Review by ggramal — APPROVED** [review]

**Pull request merged**